zscaler/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "display_name": "Zscaler",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "zscaler",
+  "metric_prefix": "zscaler.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "The Zscaler integration provides cloud security logs",
+  "guid": "595d2046-a122-4355-bfde-f4b42ece4a41",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Zscaler Integration",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
```

Comment:
"is_public" has to be set to "true" for the page to appear on the public docs integrations page.